coffeelint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "max_line_length": {
+    "value": 94,
```

Comment:
Chose a value that allowed `lib/linter-coffee-variables.coffee` to have no errors.